updatecli/updatecli.d/azcopy.yaml

```diff
@@ -15,12 +15,13 @@ scms:
 
 sources:
   latestVersion:
-    kind: shell
-    name: Get the latest `azcopy` (full) version
+    kind: githubrelease
+    name: Get the latest `azcopy` version
     spec:
-      command: bash -c 'basename "$(dirname "$(curl https://aka.ms/downloadazcopy-v10-linux --write-out "%{redirect_url}" --output /dev/null --silent --fail --show-error)" )"'
-    transformers:
-      - trimprefix: 'release-'
+      owner: Azure
+      repository: azure-storage-azcopy
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
```

Comment:
This source is missing a transformer to remove the `v` prefix: the `ARG` you changed in the `Dockerfile` does not have a `v` prefix.